### ISSUE TITLE
Selecting and unselecting text

### ISSUE THREAD

**Issue by Bibeksharma5** [nvaccess/nvda#20076]
### Is your feature request related to a problem? Please describe.

Hi NVAccess team, I want to request a small feature in NVDA. For EG: if I select any text and if I press up/down arrow once, the text will be unselected and NVDA says "unselected" followed by the previous selected text. In NVDA, there should also be an option to silence the speech of unselection message. Regards.

### Describe the solution you'd like

In NVDA document formatting settings, there should be a checkbox named "Report when the text is unselected" or something.

**Comment by CyrilleB79** [comment]
@mltony, this issue has been triaged.
Since you have worked on this in one of your add-ons, would you be interested in NVDA?
